tests/cypress/latest/plugins/index.ts

```diff
@@ -44,13 +44,14 @@ module.exports = (on: Cypress.PluginEvents, config: Cypress.PluginConfigOptions)
   config.env.aws_secret_key = process.env.AWS_SECRET_ACCESS_KEY;
   config.env.gcp_credentials = process.env.GCP_CREDENTIALS;
   config.env.gcp_project = process.env.GCP_PROJECT;
-  config.env.azure_tenant_id = process.env.AZURE_TENANT_ID
-  config.env.azure_client_id = process.env.AZURE_CLIENT_ID
-  config.env.azure_client_secret = process.env.AZURE_CLIENT_SECRET
-  config.env.azure_subscription_id = process.env.AZURE_SUBSCRIPTION_ID
+  config.env.azure_tenant_id = process.env.AZURE_TENANT_ID;
+  config.env.azure_client_id = process.env.AZURE_CLIENT_ID;
+  config.env.azure_client_secret = process.env.AZURE_CLIENT_SECRET;
+  config.env.azure_subscription_id = process.env.AZURE_SUBSCRIPTION_ID;
   config.env.skip_cluster_delete = process.env.SKIP_CLUSTER_DELETE || "false";
+  config.env.cluster_name_suffix = process.env.CLUSTER_NAME_SUFFIX || "";
```

Comment:
```suggestion
  config.env.cluster_name_suffix = process.env.CLUSTER_NAME_SUFFIX || randomstring.generate({ length: 4, capitalization: 'lowercase' });
```
Assuming this randomstring does not change every time this envvar is fetched.

I would recommend either